src/App.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import './App.css';
+import Header from './Header.js';
+import TaskList from './TaskList.js';
+import BottomButtons from './BottomButtons.js';
+import {useState} from "react";
+
+
+function App(props) {
+    const [data, setData] = useState(props.initialData);
+    const [showOnlyUncomplete, setShowOnlyUncomplete] = useState(false);
+    const [nextId, setNextId] = useState(data.length + 1);
+
+    const filteredData = data.filter(t => !t.completed);
+
+    function handleChangeField(taskId, field, value) {
+        setData(data.map(
+            t => t.id === taskId ? {...t, [field]:value} : t
+        ))
+    }
+
+    function handleItemDeleted(taskID) {
+        setData(data.filter((task) => task.id !== taskID));
+    }
+
+    function handleClearCompleted() {
+        setData(data.filter(task => !task.completed))
+    }
+
+    function handleAddTask(taskValue) {
+        for (const task of data) {
+            if (task.value === "") {
+                return;
+            }
+        }
+        const newTask = { id: "task"+String(nextId),
+                          value: taskValue,
+                          completed: false };
+        setNextId(nextId + 1);
+        setData([].concat(data, [newTask]));
+    }
+
+    const regBgColor = "white";
```

Comment:
How come you're setting actual styles in the code rather than just setting classNames?